src/plugins/intel_gpu/tests/test_utils/test_utils.cpp

```diff
@@ -388,12 +388,12 @@ std::vector<cldnn::data_types> generic_test::test_data_types() {
 double default_tolerance(data_types dt) {
     switch (dt) {
     case data_types::f16:
-        return 1e-3;
+        return 1e-2;
     case data_types::f32:
-        return 1e-5;
+        return 1e-4;
```

Comment:
For fp16 and fp32, let's not change it unless it is really necessary

Reply:
Reverted related changes.